### PR TITLE
change from sysregistries to sysregistriesv2

### PR DESCRIPTION
**Pull request by rhatdan**
We want to start supporting the registries.conf format.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@vrothberg PTAL
But I have one issue.
```
		registryPath := sysregistries.RegistriesConfPath(&types.SystemContext{SystemRegistriesConfPath: systemRegistriesConfPath})
```
There does not seem to be a replacement for this?

**Comment by vrothberg** [comment]
> There does not seem to be a replacement for this?

I think it's sufficient to tell that "no search registries are defined in the registries config". We can export the variable sysregistriesv2 in the future but how the code in `pull.go` is a bit exotic as the variable will not change after line 249.

**Comment by rhatdan** [comment]
@vrothberg Ok removed the RegistryConfPath.
Also added in blocked registries to podman info command.

**Comment by rhatdan** [comment]
@mheon @baude @giuseppe @TomSweeneyRedHat @vrothberg PTAL

**Comment by mheon** [comment]
Test failures are network flakes

**Comment by mheon** [comment]
LGTM pending green tests

File: docs/podman-info.1.md [hunk 1]
```diff
@@ -52,14 +52,14 @@ host:
...
 registries:
-  registries:
+  blocked: null
+  insecure: null
+  search:
   - quay.io
   - registry.fedoraproject.org
   - docker.io
-  - registry.access.redhat.com
+  - registry.redhat.io
```
Comment (TomSweeneyRedHat) [inline docs/podman-info.1.md]: Just a FWIW, the latest with v1.8.0 of Buildah actually has:
registries = ['docker.io', 'registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com', 'registry.centos.org']
I thought we'd changed that to get quay.io up front as shown in this exampel and registries.access.redhat.com to registry.redhat.io?

**Pull request merged**